src/harvester/queries.ts

```diff
@@ -1,67 +1,53 @@
 import { DocumentNode } from 'graphql';
 import gql from 'graphql-tag';
 
-import { Maybe, QueryDidsWithClaimsArgs } from '~/harvester/types';
+import { QueryDidsWithClaimsArgs } from '~/harvester/types';
 
 interface GraphqlQuery {
```

Comment:
This interface should be generic and take the type of the variables as a parameter